hexrd/ui/indexing/fit_grains_options_dialog.py

```diff
@@ -171,7 +175,10 @@ def update_config(self):
         self.tolerances_model.copy_to_config(config)
 
         indexing_config = HexrdConfig().indexing_config
+        indexing_config['analysis_name'] = self.working_dir
```

Comment:
I haven't tried this, but I believe the `analysis_name` should be this instead: `str(Path(self.working_dir).stem)`.

As in, it should be the stem, not the full path.

Reply:
Yes, that makes sense. Fixed.